go/database/mpt/live_trie.go

```diff
@@ -158,6 +159,32 @@ func (s *LiveTrie) VisitTrie(visitor NodeVisitor) error {
 	return s.forest.VisitTrie(&s.root, visitor)
 }
 
+// VisitAccount visits the storage nodes of an account with the given address.
+// The visited nodes contain the account node and all storage nodes of the account.
+func (s *LiveTrie) VisitAccount(address common.Address, visitor NodeVisitor) error {
```

Comment:
Based on the description I would suggest to rename this function to `VisitAccountStorage` since it is the storage that is visited, not the account.

Reply:
ok, renamed